packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -141,6 +141,12 @@ const StyledAlert = styled(Alert)`
 
 const INITIAL_FORM_VALUES = {
   name: '',
+  date: null,
+  time: '',
+  // Remove future flag check after Scheduling Beta release and replace with true as creating new release should include scheduling by default
+  isScheduled: window.strapi.future.isEnabled('contentReleasesScheduling'),
```

Comment:
I think this is not needed here, `isScheduled` when creating a release should be false always

Reply:
No, I had confirmed this with Yannis and so updated it